.github/workflows/app_interface.yaml

```diff
@@ -0,0 +1,74 @@
+---
+name: App-interface repo update
+on:
+  push:
+    branches:
+      - main
+jobs:
+  generate:
+    name: Push internal-services resources
+    runs-on: ubuntu-20.04
+    env:
+      NAMESPACE: stonesoup-int-srvc
+      REMOTE_CLIENT_CONFIG_SECRET: remote-client-config
+      REMOTE_CLIENT_CONFIG_MOUNT_PATH: "/mnt/internal-services/remote-client-config"
+      USERS: "damoreno shebert"
+      YQ_VERSION: "v4.30.8"
+    steps:
+      - name: Checkout internal-services repo
+        uses: actions/checkout@v3
+        with:
+          path: internal-services
+      - name: Checkout app-interface repo
+        uses: actions/checkout@v3
+        with:
+          path: app-interface-deployments
+          ref: main
+          repository: hacbs-release/app-interface-deployments
+          ssh-key: ${{ secrets.APP_INTERFACE_DEPLOYMENTS_SSH_PRIVATE_KEY }}
+      - name: Copy files to app-interface-deployments
+        run: |
+          mkdir -p internal-services
+          mkdir -p internal-services/crds
+          mkdir -p internal-services/manager
+          mkdir -p internal-services/rbac
+          cp ../internal-services/config/crd/bases/*.yaml internal-services/crds
+          cp ../internal-services/config/manager/manager.yaml internal-services/manager
+          cp ../internal-services/config/rbac/*.yaml internal-services/rbac
+          rm -rf internal-services/rbac/kustomization.yaml
+        working-directory: app-interface-deployments
+      - name: Setup yq
+        uses: chrisdickinson/setup-yq@latest
+        with:
+          yq-version: ${{ env.YQ_VERSION }}
+      - name: Update namespaces
+        run: |
+          find . -type f -exec sed -i 's/namespace: system/namespace: ${{ env.NAMESPACE }}/' {} \;
+        working-directory: app-interface-deployments/internal-services
+      - name: Update image
+        run: |
+          sed -i 's/image: controller:latest/image: quay.io\/redhat-appstudio\/internal-services:${{ github.event.pull_request.head.sha }}/' manager.yaml
```

Comment:
is this guaranteed to exist at this time?

Reply:
It should as the workflow runs after the commit.